### ISSUE TITLE
Set up tab: number viewpoints mixed up with number of genes

### ISSUE THREAD

**Issue by hansenp** [TheJacksonLaboratory/VPV#95]
![image](https://user-images.githubusercontent.com/10495485/32886340-afd0dea4-cac0-11e7-82c5-e984f465c525.png)

I think the number of genes should equal the number of valid gene symbols, in this case 731.

**Comment by pnrobinson** [comment]
I agree that is incorrect. The number is coming from this
```
model.getVPVGeneList().size()
```

**Comment by pnrobinson** [comment]
Owing to the switch to genemap2, VPVGenes are now creating for each gene symbol on each chromosome (gene2chromosomePosMap--we are getting the unique transcripts and a few more items).  I think we want to have this number be the number of unique gene symbols, which is accessible via
```
 public int n_totalRefGenes() { return this.geneSymbolMap.size(); } 
```
I am adding a variable in Model to store this information so we can show it in the Q/C check dialog.

**Comment by pnrobinson** [comment]
This was fixed in #100
